src/e-load-xcode.js

```diff
@@ -48,6 +51,23 @@ for (const sdk of SDK_TO_LINK) {
   childProcess.execFileSync('sudo', ['ln', '-s', sourceSDK, targetDirectory]);
 }
 
+// Unlink unnecessary macOS SDKs.
+for (const sdk of SDK_TO_UNLINK) {
+  const sourceSDK = path.resolve(macOSSDKs.path, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(sourceSDK)) continue;
+
+  const targetDirectory = path.resolve(xCodeSDKDir, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(targetDirectory)) continue;
```

Comment:
Can we check that the target is actually a symbolic link, some version of `fs.statSync()` should work